requirements.txt

```diff
@@ -7,4 +7,5 @@ pandas
 configparser
 pytest
 puppetparser
-jinja2
\ No newline at end of file
+jinja2
+glitch-python-hcl2
```

Comment:
This is fine, but it might be a good idea to include versions (perhaps the best is to create a separate issue for that). Otherwise, we might have issues in a near future related to incompatible versions.

Reply:
I will open a issue for that